reaktive-testing/src/commonMain/kotlin/com/badoo/reaktive/test/scheduler/TestScheduler.kt

```diff
@@ -53,8 +62,16 @@ class TestScheduler(
         }
 
         fun advanceBy(millis: Long) {
-            timeMillis.addAndGet(millis)
-            listeners.value.forEach { it() }
+            var currentTimerStep = millis
+            while (currentTimerStep > 0) {
+                val advance = when (minTimerStep.value) {
+                    in 1 until currentTimerStep -> minTimerStep.value
```

Comment:
if `minTimerStep` is between `1` till `millis` then keep advancing time with it, else just advance with the `millis`